cinema/serializers.py

```diff
@@ -36,13 +36,32 @@ class Meta:
         fields = ("id", "title", "description", "duration", "genres", "actors")
 
 
+class MovieImageSerializer(MovieSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
+
+
 class MovieListSerializer(MovieSerializer):
     genres = serializers.SlugRelatedField(
         many=True, read_only=True, slug_field="name"
     )
     actors = serializers.SlugRelatedField(
         many=True, read_only=True, slug_field="full_name"
     )
+    image = MovieImageSerializer(read_only=True)
```

Comment:
The `MovieListSerializer` correctly includes the `image` field, ensuring that the image URL is displayed in the movie list.